src/table/styles.js

```diff
@@ -126,13 +126,18 @@ export const VisuallyHidden = styled('span')({
 // ---------- TableTotals ----------
 
 export const TotalsCell = styled(TableCell, {
-  shouldForwardProp: (prop) => prop !== 'headerStyle' && prop !== 'isTop',
-})(({ headerStyle, isTop }) => ({
-  ...headerStyle,
+  shouldForwardProp: (prop) =>
+    prop !== 'headerStyle' && prop !== 'bodyCellStyle' && prop !== 'isTop' && prop !== 'headRowHeight',
+})(({ headerStyle, bodyCellStyle, isTop, headRowHeight }) => ({
+  ...bodyCellStyle,
   fontWeight: 'bold',
   position: 'sticky',
+  backgroundColor: headerStyle.backgroundColor,
   borderWidth: isTop ? '0px 1px 2px 0px' : '2px 1px 1px 0px',
-  top: isTop && 35,
+  // Since the shadow is applied cell by cell and shadowbox blur at cell corners spreads little bit less, the end result is
+  // a wavy shadow at bottom of the total row.
+  // boxShadow: isTop ? '0px 3px 5px -5px #a6a6a6' : '3px 0px 5px -5px #a6a6a6',
```

Comment:
Do you think we want to have a bit shadow under the Total row as the old table has ?

Reply:
Don't know but we should not spend too much time on it, since we will most likely have to change it when we get a design from UX. So no need to change anything